modules/search/src/service.rs

```diff
@@ -84,4 +84,42 @@ impl SearchService {
                 .collect(),
         })
     }
+
+    // `filters` should be of the form, "full text search({field}{op}{value})*", e.g.
+    // "some text&published>=2020/11/11&location=localhost&severity=low|high&modified=true"
+    pub async fn search_sboms<'a>(
+        &self,
+        filters: String,
+        sort: String,
+        paginated: Paginated,
+    ) -> Result<PaginatedResults<FoundSbom>, Error> {
+        let mut select = sbom::Entity::find()
```

Comment:
I've also used this pattern, and while not on your shoulders, I feel like we are repeating ourselves a lot, and perhaps @jcrossley3 could get us some more ergonomics around using search/sort.